tests/tracing/test_pruning.py

```diff
@@ -1374,7 +1373,7 @@ def test_prune_by_length(
         pytest.param(
             "pruning_skeleton",
             None,
-            1.0e-19,
+            10000,  # can assume any not-None value as the threshold is found via the IQR
```

Comment:
Useful comments like this could go in the `id=""` where they will be clearly seen if the test fails.

Reply:
I thought we'd want to keep the `id`'s short and readable but if you think it's useful I'll make this change tomorrow